hub/collections/dataset/__init__.py

```diff
@@ -132,3 +133,17 @@ def merge(datasets: Iterable[Dataset]) -> Dataset:
             assert tname not in tensors
             tensors[tname] = tvalue
     return Dataset(tensors)
+
+
+def from_tensorflow(input):
+    images, labels = [], []
+    for image, label in input.take(100):
```

Comment:
`from_tensorflow` should not be limited to 100 samples, only tests could be limited. 

As I understand this is a good beginning so here are the next steps. 
* The names of the dictionary are not considered. They should default to the name in the tfds dataset. Currently you take "data" and  "labels" as default. 
* dtags are also not considered. This will be problematic for visualization later.
* Arbitrary number of tensors
* When size of the tensors becomes bigger than Numpy can support, need to switch to dataset generators.